### ISSUE TITLE
QUESTION: Is a page going to be made for every single menu item?

### ISSUE THREAD

**Issue by JudeTeremy** [JudeJeremy/PyxisStudios#6]
Im asking this because in apps like uber and doordash you can click on the app to find more about it, similar to amazon or ebay. Im wondering if we are doing to same thing, as it would change how the website would be structured and how the "Feedback" tab would work.

**Comment by JudeJeremy** [comment]
Yeah i was also wondering about this as well. I don't it should be necessary though?
